app/commands/run_tests.py

```diff
@@ -36,7 +36,7 @@
     "--selected-tests",
     "-s",
     help="JSON string with selected tests. "
-    'Format: \'{"collection_name":{"test_suite_id":{"test_case_id": <iterations>}}}\'',
+    'Format Example: \'{"SDK YAML Tests":{"FirstChipToolSuite":{"TC-ACE-1.1": 1}}}\'',
```

Comment:
I think it's good to have the explicit fields in help, not just an example. Maybe it could have both?

Reply:
I tried both and It got confuse with too much information. I didn't managed to break the line as desired.

I changed this because the person that pointed this issue tried the following:
`{"automated_and_semi_automated":{"FirstChipToolSuite":{"TC_BINFO_1_1":1}}}`
Maybe It's not so clear to everyone what "collection_name" means

From those options, I'll leave both then. Even though it's confusing, it's better.

Reply:
Both:
<img width="567" alt="Screenshot 2024-01-25 at 09 58 07" src="https://github.com/project-chip/certification-tool-cli/assets/116589331/7af8721a-84bd-4468-9563-753b51cb55e3">

Reply:
Done in https://github.com/project-chip/certification-tool-cli/pull/9/commits/41bdf734664e263f192cd5e87ab8a7bf6e222f0b